lib/commands/-command.js

```diff
@@ -6,7 +6,7 @@ module.exports = Command.extend({
   init() {
     process.env.EMBER_CLI_ELECTRON = true;
 
-    if (this._super && this._super.init) {
+    if (this._super && this._super.init && this._super.init !== this.init) {
```

Comment:
@kevingelion are you sure you were running against the correct ember-cli version? When I change to that syntax, everything works fine for me locally. Do you have a more specific repro?

If this is a problem here, then it's probably a problem in `electron-test.js` also, so we probably want to fix it in both places, but I'd prefer to understand why this pretty fundamental mechanism of ember-cli's `CoreObject` is misbehaving for us...